nvi-commons/src/main/java/no/sikt/nva/nvi/common/model/Foundation.java

```diff
@@ -0,0 +1,55 @@
+package no.sikt.nva.nvi.common.model;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.List;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@JsonSerialize
+public record Foundation(String instanceType,
+                         Level level,
+                         List<Creator> creators,
+                         boolean international,
+                         int totalCreators) {
```

Comment:
So if we have 2 creators and 5 totalCreators, we then assume 3 is international contributors?

Reply:
Thats right, yes